metaflow/plugins/kfp/kfp.py

```diff
@@ -239,7 +240,7 @@ def _get_retries(node: DAGNode) -> Tuple[int, int]:
         return max_user_code_retries, max_user_code_retries + max_error_retries
 
     @staticmethod
-    def _get_resource_requirements(node: DAGNode) -> Dict[str, str]:
+    def _get_pod_customization(node: DAGNode) -> Dict[str, str]:
         """
         Get resource request or limit for a Metaflow step (node) set by @resources decorator.
```

Comment:
Can you update this comment to also include info that this function will get labels and annotations as well as resources with the `@resources` decorator?

Reply:
Thanks. Updated